app/src/main/java/ch/sdp/vibester/activity/MainActivity.kt

```diff
@@ -0,0 +1,40 @@
+package ch.sdp.vibester.activity
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.navigation.NavController
+import androidx.navigation.Navigation
+import androidx.navigation.ui.NavigationUI
+import androidx.navigation.ui.setupWithNavController
+import ch.sdp.vibester.R
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
```

Comment:
consider adding documentation for this new class.

Reply:
done!